CHANGES.rst

```diff
@@ -61,6 +61,7 @@ astropy.table
 
 astropy.tests
 ^^^^^^^^^^^^^
+-Deprecated function ``quantity``allclose to helper
```

Comment:
Formatting is wrong (see elsewhere in fine). Also, this is not very helpful to someone who is reading this. Write
```
The function ``quantity_allclose`` was moved to the ``units`` package with the new,
shorter name ``allclose``. It is deprecated in ``tests``. [#7252]
```

Reply:
The comments about the changelog should still be addressed.